lib/index.js

```diff
@@ -77,13 +87,46 @@ export default {
     this.subscriptions.dispose();
   },
 
+  fixJob(editor) {
+    let correctScope = false;
+
+    if (editor && editor.buffer && editor.buffer.file && editor.buffer.file.path) {
+      const { path } = editor.buffer.file;
+
+      this.baseScopes.forEach((scope) => {
+        const extension = scope.replace('source', '');
+
+        if (path.lastIndexOf(extension) === path.length - extension.length) {
+          correctScope = true;
+        }
+      });
+    }
+
+    if (!correctScope) {
+      return;
+    }
+
+    const oldFileContent = editor.getText();
+    const cursorPosition = editor.getCursorBufferPosition();
```

Comment:
The way you currently have this written it will automatically consolidate all `Cursor`s down to a single one at the position of the last added one when fixing the document (regardless of whether anything changed!).

I really like the idea of attempting to restore the cursor position, so here are two ways I see of fixing this to support multiple cursors:

* The quick way of just leaving it as is, and forcing users to accept it
* The "right" way of tracking all cursors and setting the position of each one


If we want to track all the `Cursor`s I _think_ it would look something like:
1. Grab all the `Cursor`s with [TextEditor::getCursors](https://atom.io/docs/api/v1.26.1/TextEditor#instance-getCursors)
2. Store them in a `Map`, with the `Cursor` as the key, and the current position as the value
3. Run the fix job
4. Attempt to set each `Cursor`'s position to the cached position

A potential issue I see with this is what will Atom do to the existing `Cursor`s when we call `editor.setText`? If it consolidates them together then, _destorying_ the old ones, we may need to create new `Cursor`'s at the old locations in the older they were added which would complicate things further.